APMrover2/Rover.h

```diff
@@ -387,6 +387,16 @@ class Rover : public AP_HAL::HAL::Callbacks {
         LowPassFilterFloat throttle_filt = LowPassFilterFloat(2.0f);
     } cruise_learn;
 
+    // sailboat variables
+    enum Sailboat_Tack {
+        Tack_Port,
+        Tack_STBD
+    };
+    bool _sailboat_tacking;                 // true when sailboat is in the process of tacking to a new heading
```

Comment:
yes, I totally agree.  I've moved into a structure for now.  I think a class is probably the best way to go as well but instead of doing that now I will just create an issue for it.  Potentially the balance bot should also be moved into a class.